spec/github_helper_spec.rb

```diff
@@ -47,6 +47,25 @@
     end
   end
 
+  describe 'github_client' do
+    let(:client) do
+      instance_double(
+        Octokit::Client,
+        user: instance_double('User', name: 'test'),
+        'auto_paginate=': nil
+      )
+    end
+
+    before do
+      allow(described_class).to receive(:github_token!).and_return('')
+      allow(Octokit::Client).to receive(:new).and_return(client)
+    end
+
+    it 'is not nil' do
+      expect(described_class.github_client).not_to be_nil
+    end
+  end
```

Comment:
I suggest to add another test checking that the memoization is working as expected. Especially than when calling the `github_client` multiple times, it will only create a single `Octokit::Client` instance and only authenticate once (instead of each time it's referenced)

Reply:
This is probably a good idea – I'll take a look at it soon!